sentry-spring-jakarta/src/main/java/io/sentry/spring/jakarta/webflux/ReactorUtils.java

```diff
@@ -87,20 +92,20 @@ public static <T> Flux<T> withSentry(final @NotNull Flux<T> flux) {
    */
   @ApiStatus.Experimental
   public static <T> Flux<T> withSentryNewMainHubClone(final @NotNull Flux<T> flux) {
-    final @NotNull IHub hub = Sentry.cloneMainHub();
+    final @NotNull IScopes hub = Sentry.cloneMainHub();
```

Comment:
Rename to `scopes`

Reply:
Changed in https://github.com/getsentry/sentry-java/pull/3366